deploy/gallery/create_service.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+# This script creates the required linked service & the spark 
+# job definition on the Synapse workspace and should be run 
+# prior to executing the pipeline template from Synapse gallery.
+
+set -e
+
+ENV_CODE=${1:-${ENV_CODE}}
+
+if [[ -z ${ENV_CODE} ]]
+  then
+    echo "Missing Environent Code"
+    exit 1
+fi
+
+base_dir="$(cd "$(dirname "$0")/../../" && pwd)"
+
+# Uncompress the custom vision model v2 package
+if ! unzip "${base_dir}"/custom-vision-model-v2.zip -d "${base_dir}"/custom-vision-model-v2-"${ENV_CODE}";
```

Comment:
Are we building on install+configure+package / setup script? If so, I would suggest we first add a check for presence of zip file and then continue. If zip file is not present, we prompt the user to run the package.sh before running this script.